include/common.h

```diff
@@ -978,6 +978,7 @@ extern s32 D_800F4830[];
 extern u16 D_800F4776[];
 extern void (*D_800F3134[])(struct BackgroundObj* arg0);
 extern struct Prim D_800EE504[];
+extern void (*D_800F2AD4[1])(struct QuxObj*);
```

Comment:
I've been putting these in `func_tables.h` next to the matching comment

Reply:
Ok moved